src/mapml/handlers/ContextMenu.js

```diff
@@ -194,6 +203,12 @@ export var ContextMenu = L.Handler.extend({
     context._layerClicked.layer._layerEl.focus();
   },
 
+  _copyLayer: function (e) {
+    let context = e instanceof KeyboardEvent ? this._map.contextMenu : this.contextMenu,
+      layerElem = context._layerClicked.layer._layerEl;
+    context._copyData(layerElem.outerHTML);
```

Comment:
I'm curious why the _copyData is required.  I realize that it's not your code, and it may be inherited from the context menu, but I noticed that the MDN docs suggest what it does is deprecated and can be replaced with the [Clipboard API](https://developer.mozilla.org/en-US/docs/Web/API/Document/execCommand)